docs/root/start/building/building.rst

```diff
@@ -5,27 +5,29 @@ Building
 
 .. _building_requirements:
 
-In order to compile the artifacts used by the Envoy Mobile library,
-your system must also be set up for building Envoy. To get started, you can use
-`this quick start guide
+To build Envoy Mobile, your system must also be set up for building Envoy.
+To get started, you can use `this quick start guide
 <https://github.com/envoyproxy/envoy/tree/master/bazel#quick-start-bazel-build-for-developers>`_.
 
 Ensure that the ``envoy`` **submodule** is initialized when cloning by using ``--recursive``:
 
 ``git clone https://github.com/lyft/envoy-mobile.git --recursive``
 
-If the repo was not initially cloned recursively, initialize the Envoy
-submodule with ``git submodule update --init``.
+If the repo was not initially cloned recursively, you can manually initialize the Envoy submodule:
+
+``git submodule update --init``
 
 ------------------
 Bazel requirements
 ------------------
 
-Envoy-Mobile is compiled using Bazel 0.28.0.
+Envoy Mobile is compiled using Bazel 0.29.1.
```

Comment:
Maybe instead of printing the version here we can link to https://github.com/lyft/envoy-mobile/blob/master/.bazelversion